tests/integration/targets/dcnm_vrf/tests/dcnm/self-contained-tests/vrf_lite.yaml

```diff
@@ -0,0 +1,286 @@
+##############################################
+##                 SETUP                    ##
+##############################################
+
+- set_fact:
+    rest_path: "/rest/control/fabrics/{{ test_fabric }}"
+  when: controller_version == "11"
+
+- set_fact:
+    rest_path: "/appcenter/cisco/ndfc/api/v1/lan-fabric/rest/control/fabrics/{{ test_fabric }}"
+  when: controller_version >= "12"
+
+- name: MERGED - Verify if fabric is deployed.
+  cisco.dcnm.dcnm_rest:
+    method: GET
+    path: "{{ rest_path }}"
+  register: result
+
+- assert:
+    that:
+    - 'result.response.DATA != None'
+
+- name: MERGED - Clean up any existing vrfs
+  cisco.dcnm.dcnm_vrf:
+    fabric: "{{ test_fabric }}"
+    state: deleted
+
+- name: VRF LITE - sleep for 40 seconds for DCNM to completely update the state
```

Comment:
Use `retry` block

https://github.com/CiscoDevNet/ansible-dcnm/blob/develop/tests/integration/targets/dcnm_vrf/tests/dcnm/merged.yaml#L50

Reply:
This can be removed after check_deploy support, which is planned for a later release.